docs/Actions/send-email.md

```diff
@@ -0,0 +1,531 @@
+            "Value": "",
+            "IsExpression": false
+        },
+        "FileField4": {
+            "Expression": "",
+            "Value": "",
+            "IsExpression": false
+        },
+        "FileField5": {
+            "Expression": "",
+            "Value": "",
+            "IsExpression": false
+        },
+        "FileToken1": "",
+        "IgnoreErros": ""
+    }
+}
+```
+
+### `3. A more coplex setup for sending a email`
+
+In this example we display how to use a mix of text and tokens to set parameters. You can also set the From parameter email display name as in this example: `<The Sender of the Email>support@plantanapp.com`. In addition we setup an token to the logo so we can send it with the email as attachment.  
+
+```json
+{
+    "Title": "Execute Actions",
+    "ActionType": "ExecuteActions",
+    "Description": "A more coplex setup for sending a email",
+    "Condition": null,
+    "Parameters": {
+        "ActionList": [
+            {
+                "Id": -1,
+                "$_uid": "action15918762609049277",
+                "Parameters": {
+                    "Data": [
+                        {
+                            "value": "/Images/logo.png",
+                            "name": "Path"
+                        }
+                    ]
+                },
+                "ActionType": "InjectData",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "Context"
+                    }
+                }
+            },
+            {
+                "Title": "Send Email",
+                "ActionType": "SendMail",
+                "Description": "Complex example",
+                "Condition": null,
+                "Parameters": {
+                    "From": "<The Sender of the Email>support@plantanapp.com",
+                    "DetermineEmail": "",
+                    "To": "user1@plantanapp.com; user2@plantanapp.com; [User:Email]",
+                    "ToAllUsers": "",
+                    "ReplyTo": "",
+                    "Cc": "user3@plantanapp.com; user4@plantanapp.com;",
+                    "Bcc": "user5@plantanapp.com; user6@plantanapp.com;",
+                    "Subject": "Multiple Email sent",
+                    "Format": {
+                        "Expression": "",
+                        "Value": "Html",
+                        "IsExpression": false,
+                        "Parameters": {}
+                    },
+                    "DnnEmailTemplate": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false,
+                        "Parameters": {}
+                    },
+                    "Body": "<p>Dear Customers,</p><p>This is an example using tokens. The email was sent to&nbsp;</p>",

```

Comment:
This statement looks unfinished. Feels like there should have been a list of recipients?

Reply:
fixed